### PR TITLE
Update team-meetings.md

### PR DESCRIPTION
**Pull request by stevenino**
Updating with the new meeting cadence and the updated descriptions of the meetings along with the facilitator role.

## Summary of Changes
Updating with the new meeting cadence and the updated descriptions of the meetings along with the facilitator role.

### CODE REVIEW

File: docs/How-We-Work/team-meetings.md [hunk 1]
```diff
@@ -40,58 +40,69 @@ A typical sprint schedule is described in the table below.
...
-- **Attendees**: Core team 
-- **Facilitator**: Raft PM/Scrum Master
-- **When**: Daily 12pm-12:15pm EST and asynchronously on Thursdays and every other Tuesday when the team has sprint ceremonies
-- **Format**: Each team member gives quick update to answer these questions 
-  - Facilitator shares their screen and pulls up assigned tickets for each team member in the current sprint. 
+- **Goal:** To help align the team and share progress, stay accountable, and resolve potential dependencies and blockers quickly.
+- **Attendees:** Core team 
+- **Facilitator:** Raft PM/Scrum Master
+- **When:** Daily 12pm-12:15pm EST and asynchronously on Thursdays and every other Tuesday when the team has sprint ceremonies
+- **Format:** Each team member gives an brief and intentional update to answer these questions
```
Comment (lfrohlich) [inline docs/How-We-Work/team-meetings.md:47]:
```suggestion
- **Format:** Each team member gives a brief and intentional update to answer these questions 
```

File: docs/How-We-Work/team-meetings.md [hunk 2]
```diff
@@ -40,58 +40,69 @@ A typical sprint schedule is described in the table below.
...
+ ### Sprint Review 
+- **Goal:** To review the work that was completed in the last two weeks and identify work that will roll over into the next sprint cycle. 
 - **Attendees:** Core team 
 - **Facilitator:** Raft PM/Scrum Master
-- **When:** Every other Tuesday 10:30am-12pm EST (combined with Backlog Refinement) 
-- **Format:** Product owner should come with priority issues prepared and ready to discuss with the rest of the team. Dev and UX leads should come with refined tickets and should be prepared with updates on the current issues. Work in progress should be assessed and the team should decide if work should continue into the new sprint.
+- **When:** Every other Tuesday 10:00am-12pm EST (combined with Internal Demo, Retro and Sprint Planning) 
+- **Format:** The team will start by reviewing/updating the Sprint Board. We will discuss closed/merged tickets and work in progress tickets to assess thier status for the upcoming sprint. Key metrics such as velocity will also be analyzed. By the end of the meeting the team will have a clear undersatnding of what tickets will continue to be worked in the next sprint what finished products will be demoed to the team.
```
Comment (lfrohlich) [inline docs/How-We-Work/team-meetings.md:69]:
```suggestion
- **Format:** The team will start by reviewing/updating the Sprint Board. We will discuss closed/merged tickets and work in progress tickets to assess their status for the upcoming sprint. Key metrics such as velocity will also be analyzed. By the end of the meeting, the team will have a clear understanding of which tickets will continue to be worked in the next sprint and which finished products will be demoed to the team.
```

File: docs/How-We-Work/team-meetings.md [hunk 3]
```diff
@@ -40,58 +40,69 @@ A typical sprint schedule is described in the table below.
...
+  - Whoever facilitates the retro is responsible for documenting commitments in GH issues or Team Charter as appropriate. Mural board links can be found [here](https://hhsgov.sharepoint.com/sites/TANFDataPortalOFA/_layouts/15/Doc.aspx?sourcedoc={cbce2e75-17b2-4e70-b422-60d034fcd4af}&action=edit&wd=target%28General.one%7C76fa641e-4d01-436b-a079-42ae19270071%2FGeneral%20Links%7C38eda373-ae66-41ed-9d2d-e7b010d5b312%2F%29)
+ 
+### Sprint Planning 
+- **Goal:** To define, assign, and point the work we’re committing to complete in the next sprint cycle 
+- **Attendees:** Core team 
+- **Facilitator:** Raft PM/Scrum Master
+- **When:** Every other Tuesday 10:00am-12pm EST (combined with Sprint Review, Internal Demo, and Retro) 
+- **Format:** Product owner and raft product manager should come prepared with refined priority issues for the team to plan into the sprint. TFor better planning, the team will calculate their capacity and point the requested tickets before they pull the work the sprint. At the end of the meeting the team should have a clear understanding and feel comfortable about completing the new work and rollover work they commited to.
```
Comment (lfrohlich) [inline docs/How-We-Work/team-meetings.md:97]:
```suggestion
- **Format:** Product owner and raft product manager should come prepared with refined priority issues for the team to plan into the sprint. For better planning, the team will calculate their capacity and point the requested tickets before they pull the work into the sprint. At the end of the meeting the team should have a clear understanding and feel comfortable about completing the new work and rollover work they committed to.
```

File: docs/How-We-Work/team-meetings.md [hunk 4]
```diff
@@ -40,58 +40,69 @@ A typical sprint schedule is described in the table below.
...
+- **When:** Every Tuesday on non-sprint planning weeks 11:00am-12:00pm EST / Every Tuesday on sprint planning weeks 3:00pm-4:00pm EST
+- **Format:** The product owner and raft product manager will collaborate with the leads to review the backlog and prioritize issues in support of the next release and make sure items are ready for the upcoming Sprint Planning session. By the end of the session, there will be a common understating of the upcoming priorities and acceptance criteria of refined issues. The refined set of issues should be sent to the sprint board before Sprint Planning. If there are outstanding questions on a particular issue they will be noted within the unrefined issue and remain in the backlog. The agenda and notes can be found in the [Product Notebook](https://hhsgov.sharepoint.com/sites/TANFDataPortalOFA/_layouts/15/Doc.aspx?sourcedoc={cbce2e75-17b2-4e70-b422-60d034fcd4af}&action=edit&wd=target%28Product.one%7Ccfbcc7fb-4b00-4c43-9e29-70bdedd83b98%2FBacklog%20Refinement%7C4ef1b64b-327d-4628-823a-0d1fc5fce6ea%2F%29) within the TDP OneNote.
```
Comment (lfrohlich) [inline docs/How-We-Work/team-meetings.md:62]:
```suggestion
- **Format:** The product owner and raft product manager will collaborate with the leads to review the backlog and prioritize issues in support of the next release and make sure items are ready for the upcoming Sprint Planning session. By the end of the session, there will be a common understanding of the upcoming priorities and acceptance criteria of refined issues. The refined set of issues should be sent to the sprint board before Sprint Planning. If there are outstanding questions on a particular issue they will be noted within the unrefined issue and remain in the backlog. The agenda and notes can be found in the [Product Notebook](https://hhsgov.sharepoint.com/sites/TANFDataPortalOFA/_layouts/15/Doc.aspx?sourcedoc={cbce2e75-17b2-4e70-b422-60d034fcd4af}&action=edit&wd=target%28Product.one%7Ccfbcc7fb-4b00-4c43-9e29-70bdedd83b98%2FBacklog%20Refinement%7C4ef1b64b-327d-4628-823a-0d1fc5fce6ea%2F%29) within the TDP OneNote.
```

File: docs/How-We-Work/team-meetings.md [hunk 5]
```diff
@@ -40,58 +40,69 @@ A typical sprint schedule is described in the table below.
...
+- **Attendees:** Core team 
+- **Facilitator:** Raft PM/Scrum Master
+- **When:** Every other Tuesday 10:00am-12pm EST (combined with Sprint Review, Internal Demo, and Retro) 
+- **Format:** Product owner and raft product manager should come prepared with refined priority issues for the team to plan into the sprint. For better planning, the team will calculate their capacity and point the requested tickets before they pull the work into the sprint. At the end of the meeting the team should have a clear understanding and feel comfortable about completing the new work and rollover work they committed to.
+
+### Sprint Demo 
+- **Goal:** To showcase the work completed in the sprint and demo new features to other stakeholders. The Sprint Demo is a more formal version of the internl demo. The issues that are demoed have received product owner’s approval in the internal demo. 
+- **Attendees**: Core team, TANF Data team (Thomas, Yun, Kathleen), and other stakeholders by invitation.  
+- **Facilitator:** Raft PM/Scrum Master
+- **When:** Every other Tuesday 1:15pm-1:45pm
```
Comment (lfrohlich) [inline docs/How-We-Work/team-meetings.md:103]:
```suggestion
- **When:** Every other Tuesday 2:00pm-2:30pm EST, as needed
```

File: docs/How-We-Work/team-meetings.md [hunk 6]
```diff
@@ -40,58 +40,69 @@ A typical sprint schedule is described in the table below.
...
+- **Format:** This meeting is a formal version of internal demo with an emphasis on demonstrating work to stakeholders outside the product team. This demo should emphasize completed work that has a direct impact to the end user, it should not be a status of the work that has been done. The product owner or her designee will demo new work and features to the attendees. How attendees give feedback on the demo needs to be determined.
```
Comment (lfrohlich) [inline docs/How-We-Work/team-meetings.md:104]:
```suggestion
- **Format:** This meeting is a formal version of internal demo with an emphasis on demonstrating work to stakeholders outside the product team. This demo should emphasize completed work that has a direct impact to the end user, it should not be a status of the work that has been done. The author of the feature will demo new work and features to the attendees. How attendees give feedback on the demo needs to be determined. 
```

File: docs/How-We-Work/team-meetings.md [hunk 7]
```diff
@@ -110,16 +121,16 @@ A typical sprint schedule is described in the table below.
...
-* **Facilitator:** OFA PO
+* **Facilitator:** Raft PM/Scrum Master
 * **When:** Every other Friday 10am-11am ET
-* **Format:** A link to the meeting [agenda](https://hhsgov.sharepoint.com/:o:/r/sites/TANFDataPortalOFA/SiteAssets/TANF%20Data%20Portal%20(OFA)%20Notebook?d=wcbce2e7517b24e70b42260d034fcd4af&csf=1&web=1&e=1mTgNd) (in TDP OneNote) is included in the meeting invite. All attendees are invited to add to the agenda prior to the meeting. Notes and action items will be captured within the meeting agenda link.
+* **Format:** The agenda can be found in the [Product Notebook](https://hhsgov.sharepoint.com/sites/TANFDataPortalOFA/_layouts/15/Doc.aspx?sourcedoc={cbce2e75-17b2-4e70-b422-60d034fcd4af}&action=edit&wd=target%28Product.one%7Ccfbcc7fb-4b00-4c43-9e29-70bdedd83b98%2FCross%20Product%20Sync%7Cec320778-42b6-43dc-af81-72494962f7bb%2F%29) within the TDP OneNote. Notes and action items are kept here as well. All attendees are invited to add to the agenda prior to the meeting. Raft and OFA Product Team use this time as a weekly discussion and debrief. Participants discuss the status and updates to the roadmap, releases, high priority items.
```
Comment (lfrohlich) [inline docs/How-We-Work/team-meetings.md:133]:
```suggestion
* **Format:** The agenda can be found in the [Product Notebook](https://hhsgov.sharepoint.com/sites/TANFDataPortalOFA/_layouts/15/Doc.aspx?sourcedoc={cbce2e75-17b2-4e70-b422-60d034fcd4af}&action=edit&wd=target%28Product.one%7Ccfbcc7fb-4b00-4c43-9e29-70bdedd83b98%2FCross%20Product%20Sync%7Cec320778-42b6-43dc-af81-72494962f7bb%2F%29) within the TDP OneNote. Notes and action items are kept here as well. All attendees are invited to add to the agenda prior to the meeting. Raft and OFA Product Team use this time as a biweekly discussion and debrief. Participants discuss the status and updates to the roadmap, releases, high priority items.
```

**Review by lfrohlich — APPROVED** [review]
Fixed some typos and made some clarifications throughout.

File: docs/How-We-Work/team-meetings.md [hunk 8]
```diff
@@ -40,58 +40,69 @@ A typical sprint schedule is described in the table below.
...
+- **When:**  Every other Tuesday 10:00am-12pm EST (combined with Sprint Review, Retro and Sprint Planning)
+- **Format:** The team will start by demoing completed work that was identified during sprint review. By the end of the meeting, we will have an apprecition of the work that was completed in the last two weeks. The team should decide what is ready to be demoed (and label it as “demo”), who will conduct the demo/s, and if special guests should be invited to attend the larger demo event later in the day. All demoed issues must be merged first. Facilitator will take note of what issues will be demoed and who will be presenting for each issue. OFA Product Owner and OFA Tech Lead will sign off on Raft's recommendation of issues to be demoed.
```
Comment (ADPennington) [inline docs/How-We-Work/team-meetings.md:76]:
```suggestion
- **Format:** The team will start by demoing completed work that was identified during sprint review. By the end of the meeting, we will have an appreciation of the work that was completed in the last two weeks. The team should decide what is ready to be demoed (and label it as “demo”), who will conduct the demo/s, and if special guests should be invited to attend the larger demo event later in the day. All demoed issues must be merged first. Facilitator will take note of what issues will be demoed and who will be presenting for each issue. OFA Product Owner and OFA Tech Lead will sign off on Raft's recommendation of issues to be demoed.
```

**Review by ADPennington — CHANGES_REQUESTED** [review]
looks great @stevenino. just a few suggestions from @lfrohlich and me.

**Comment by stevenino** [comment]
> looks great @stevenino. just a few suggestions from @lfrohlich and me.

Thanks @ADPennington. I accepted all of the suggestions.

**Review by ADPennington — COMMENTED** [review]
👍🏾

**Review by abottoms-coder — APPROVED** [review]

**Pull request merged**